### PR TITLE
add proxy and sample thrift and fix some minor issues

### PR DESCRIPTION
**Pull request by SK2iP**

### CODE REVIEW

File: docs/2-sampleserver/0-sample-python-base-code-gRPC/index.md [hunk 1]
```diff
@@ -27,7 +28,8 @@ cd sample-playmaker-server-python-grpc
 # Activate venv/anaconda before this step!
 pip install -r requirements.txt
 
-./generate.sh # Generate the gRPC files
+./generate.sh
```
Comment (naderzare) [inline docs/2-sampleserver/0-sample-python-base-code-gRPC/index.md]: comment should be in same line or the line before

File: docs/2-sampleserver/1-sample-python-base-code-thrift/index.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+- `-t team_name`: Specify the team name.
+- `--rpc-port PORT`: Specify the RPC port (default: 50051).
+- `-d`: Enable debug mode.
+
+Launch the opponent team, start the monitor app image. press <kbd>Ctrl</kbd> + <kbd>C</kbd> to connect to the server, and <kbd>Ctrl</kbd> + <kbd>K</kbd> for kick-off!
+
+### Tutorial Video (English)
+
+TDB
```
Comment (naderzare) [inline docs/2-sampleserver/1-sample-python-base-code-thrift/index.md]: add link of the new english video

File: static/data/team.json [hunk 3]
```diff
@@ -21,15 +21,17 @@
...
+      "linkedin": "https://www.linkedin.com/in/sadra-khanjari/",
+      "img": "https://avatars.githubusercontent.com/u/45975703?v=4",
+      "github": "https://github.com/SK2iP"
     },
     {
       "name": "Arad Firouzkouhi",
-      "email": "",
-      "linkedin": "",
-      "github": ""
+      "email": "aradfir@gmail.com",
```
Comment (naderzare) [inline static/data/team.json:31]: are you ARAD?

**Review by naderzare — CHANGES_REQUESTED** [review]

File: static/data/team.json (hunk 3 above)
Comment (SK2iP) [reply static/data/team.json:31]: Last night, Arad submitted a pull request with his profile, and I accepted it. Then, I pulled from main.

File: docs/2-sampleserver/1-sample-python-base-code-thrift/index.md (hunk 2 above)
Comment (SK2iP) [reply docs/2-sampleserver/1-sample-python-base-code-thrift/index.md]: fixed it

**Comment by SK2iP** [comment]
I have fixed everything you requested. Please check the pull request and review it again.
@naderzare

**Pull request merged**